panda.config.ts

```diff
@@ -148,8 +148,9 @@ const globalCss = defineGlobalStyles({
         userSelect: 'none',
       },
     },
-  },
-  'html, body, #root, #app': { height: '100%', fontSize: '16px' },
+  }, // eslint-disable-next-line
```

Comment:
In order to disable a lint rule, a couple things are required:

1. Disable rules explicitly by name. Lone `// eslint-disable-next-line` that disables all rules is not allowed.
2. Include an inline comment that explains why disabling the linter is necessary/reasonable.

In general, disabling the linter is not recommended, and will only be allowed if a sufficient argument is made.